miqa/learning/nn_classifier.py

```diff
@@ -282,9 +287,20 @@ def evaluate1(model_path, image_path):
     print(f'Network output: {result}')
     print(f'Overall quality of {image_path}, on 0-10 scale: {result[0]:.1f}')
 
+    labeled_results = {
+        'overall_quality': result[0],
+        'signal_noise_ratio': result[1],
+        'contrast_noise_ratio': result[2],
+    }
+    for index, artifact_name in enumerate(artifacts):
+        labeled_results[artifact_name] = result[index + 3]
+    return labeled_results
+
 
 class CombinedLoss(torch.nn.Module):
-    def __init__(self, binary_class_weights, focal_loss=monai.losses.FocalLoss()):
+    default_focal_loss = monai.losses.FocalLoss()
+
+    def __init__(self, binary_class_weights, focal_loss=default_focal_loss):
```

Comment:
In Python, [we should avoid mutable default arguments](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).

```python
def __init__(self, binary_class_weights, focal_loss=None):
    if focal_loss is None:
        focal_loss = monai.losses.FocalLoss()